commands/operator-sdk/cmd/scorecard/basic_tests.go

```diff
@@ -0,0 +1,207 @@
+	for k, v := range obj.Object["status"].(map[string]interface{}) {
+		statCopy[k] = v
+	}
+	specMap := obj.Object["spec"].(map[string]interface{})
+	err = modifySpecAndCheck(specMap, obj)
+	if err != nil {
+		test.earnedPoints = 0
+		scTests = append(scTests, test)
+		return nil
+	}
+	test.earnedPoints = 1
+	scTests = append(scTests, test)
+	return nil
+}
+
+// modifySpecAndCheck is a helper function for checkStatusUpdate
+func modifySpecAndCheck(specMap map[string]interface{}, obj unstructured.Unstructured) error {
+	statCopy := make(map[string]interface{})
+	for k, v := range obj.Object["status"].(map[string]interface{}) {
+		statCopy[k] = v
+	}
+	var err error
+	for k, v := range specMap {
+		mapType := false
+		switch t := v.(type) {
+		case int64:
+			specMap[k] = specMap[k].(int64) + 1
+		case float64:
+			specMap[k] = specMap[k].(float64) + 1
+		case string:
+			// TODO: try and find out how to make this better
+			// Since strings may be very operator specific, this test may not work correctly in many cases
+			specMap[k] = fmt.Sprintf("operator sdk test value %f", rand.Float64())
+		case bool:
+			specMap[k] = !specMap[k].(bool)
+		case map[string]interface{}:
+			mapType = true
+			err = modifySpecAndCheck(specMap[k].(map[string]interface{}), obj)
+		case []map[string]interface{}:
+			mapType = true
+			for _, item := range specMap[k].([]map[string]interface{}) {
+				err = modifySpecAndCheck(item, obj)
+				if err != nil {
+					break
+				}
+			}
+		case []interface{}: // TODO: Decide how this should be handled
+		case interface{}: // same as above
+		default:
+			fmt.Printf("Unknown type for key (%s) in spec: (%v)\n", k, reflect.TypeOf(t))
+		}
+		if !mapType {
+			runtimeClient.Update(context.TODO(), &obj)
+			err = wait.Poll(time.Second*1, time.Second*15, func() (done bool, err error) {
+				runtimeClient.Get(context.TODO(), types.NamespacedName{Namespace: SCConf.Namespace, Name: name}, &obj)
+				if err != nil {
+					return false, err
+				}
+				return !reflect.DeepEqual(statCopy, obj.Object["status"]), nil
+			})
+		}
+		if err != nil {
+			return err
+		}
+		//reset stat copy to match
+		statCopy = make(map[string]interface{})
+		for k, v := range obj.Object["status"].(map[string]interface{}) {
+			statCopy[k] = v
+		}
+	}
+	return nil
+}
+
+// wiritingIntoCRsHasEffect simply looks at the proxy logs and verifies that the operator is sending PUT
+// and/or POST requests to the API server, which should mean that it is creating or modifying resources.
+func writingIntoCRsHasEffect(obj unstructured.Unstructured) (string, error) {
+	test := scorecardTest{testType: basicOperator, name: "Writing into CRs has an effect", maximumPoints: 1}
+	kubeclient, err := kubernetes.NewForConfig(kubeconfig)
+	if err != nil {
+		return "", fmt.Errorf("failed to create kubeclient: %v", err)
+	}
+	dep := &appsv1.Deployment{}
+	err = runtimeClient.Get(context.TODO(), types.NamespacedName{Namespace: SCConf.Namespace, Name: deploymentName}, dep)
```

Comment:
Pass `deploymentName` as an arg. And check that the global var is set after `createFromYamlFile()` before passing it in.